src/devices/wholeBodyDynamics/SixAxisForceTorqueMeasureHelpers.cpp

```diff
@@ -32,13 +46,41 @@ const iDynTree::Wrench& SixAxisForceTorqueMeasureProcessor::offset() const
     return m_offset;
 }
 
+iDynTree::Wrench& SixAxisForceTorqueMeasureProcessor::estimatedOffset()
+{
+    return m_estimated_offset;
+}
```

Comment:
As this parameters is not used at all in this class, I suggest to move it outside of the other class, for example as a vector of iDynTree::Wrench in https://github.com/robotology/codyco-modules/pull/281/files#diff-a54e123540c7c6eb91d20ac92449e879R612 .

Reply:
Probably the wrong approach, but It made sense to me that each sensor processor keeps track of both the offset preEstimated or otherwise. 
Would it make more sense to do the offset replacement of https://github.com/robotology/codyco-modules/blob/0926d6f9ccb2a6d43a800a4daeb78a9e230062cb/src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp#L2399 inside the processor itself?